Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/common/ClusterManagerEx.java

```diff
@@ -69,6 +74,54 @@ public static ClusterManagerEx getInstance() {
         return instance;
     }
 
+    public String getClusterConnectionString(final @NotNull String clusterName) {
+        if (isAzureChinaEnvironment()) {
+            return String.format("https://%s.azurehdinsight.cn", clusterName);
+        } else {
+            return String.format("https://%s.azurehdinsight.net", clusterName);
+        }
+    }
+
+    public String getBlobFullName(@NotNull final String storageName) {
+        return storageName + getBlobSuffix();
+    }
+
+    public String getBlobSuffix() {
+        if (isAzureChinaEnvironment()){
+            return ".blob.core.chinacloudapi.cn";
+        } else {
+            return "blob.core.windows.net";
```

Comment:
Is the dot `.` missing at the beginning?

Reply:
Fixed.